packages/core/src/context/ContextSystem.ts

```diff
@@ -278,5 +293,22 @@ export class ContextSystem extends System
 
             /* eslint-enable max-len */
         }
+
+        const hasuint32
+            = (gl instanceof WebGL2RenderingContext)
+            || !!(gl as WebGLRenderingContext).getExtension('OES_element_index_uint');
+
+        this.supports.uint32Indices = hasuint32;
+
+        if (!hasuint32)
+        {
+            /* eslint-disable max-len */
+
+            /* eslint-disable no-console */
```

Comment:
You can combine these two disable lines like this:

/* eslint-disable max-len, no-console */

Reply:
> Much better!
> 
> One small thing: Can you make the doc `@private` for supports object? It’s only used internally for now, would like to not make new public APIs if we don’t need to.

Hmm, i think it as should be public in a future. Some as `webGLVersion`.